src/views/package.json

```diff
@@ -36,5 +36,10 @@
     "ts-loader": "^4.5.0",
     "webpack": "^4.44.1"
   },
-  "dependencies": {}
-}
+  "dependencies": {},
+  "overrides": {
+    "qs": "6.10.3",
+    "async": "3.2.3",
```

Comment:
Should this (and `qs` above) be `^` or `~` prefixed?

Reply:
Sure. I can do that